community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/outputs.tf

```diff
@@ -20,4 +20,20 @@ output "nodeset_tpu" {
     condition     = (var.node_type == null) != (var.accelerator_config == { topology : "", version : "" })
     error_message = "Either a node_type or an accelerator_config must be provided."
   }
+
+  precondition {
+    condition     = ((local.tpu_core_count / 8) <= var.node_count_dynamic_max) || ((local.tpu_core_count / 8) <= var.node_count_static)
+    error_message = <<-EOD
+      When using TPUs there should be at least one node per every 8 cores. 
+      Currently there are ${local.tpu_core_count} cores but only ${var.node_count_static} static nodes and ${var.node_count_dynamic_max} dynamic nodes.
+    EOD
+  }
+
+  precondition {
+    condition     = (var.node_count_dynamic_max % (local.tpu_core_count / 8) == 0) && (var.node_count_static % (local.tpu_core_count / 8) == 0)
+    error_message = <<-EOD
+      In order to use all the nodes in this deployment, the number of worker nodes should be a multiple of ${local.tpu_core_count / 8}.
+      This will prevent nodes from not having a TPU host for job scheduling.
```

Comment:
s/will prevent nodes from not having/will prevent nodes from having

Reply:
->will prevent nodes from having

This would imply that the condition is to preventing the nodes from having a TPU host right? Please re-open this if I misunderstood the comment.